packages/components/components/elvis-stepper/src/react/StepContent.tsx

```diff
@@ -38,14 +38,22 @@ export const StepContent: FC<StepContentProps> = function ({
         ) : (
           <div></div>
         )}
-        <PrimaryButton
-          disabled={!isReachable(isForced, currentStep + 1, steps)}
-          onClick={() =>
-            handleStepChange(isReachable(isForced, currentStep + 1, steps) ? currentStep + 1 : currentStep)
-          }
-        >
-          {steps?.[currentStep]?.nextButtonText ?? 'Neste'}
-        </PrimaryButton>
+        {steps?.[currentStep]?.nextButtonState === 'loading' ? (
+          <PrimaryButton isLoading disabled={!isReachable(isForced, currentStep + 1, steps)}>
```

Comment:
I think we can add this to the button 😄
```
aria-disabled={isLoading}
aria-label={isLoading ? "Laster inn" : ""}
```
I saw it here: https://www.bekk.christmas/post/2023/24/accessible-loading-button

Reply:
Nice!

Reply:
I saw that I had a few typos in the original comment, so I updated it.